### DISCUSSION TITLE
osu steals my tablet from the OS until i unplug it/ plug it back in

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#14579] [Q&A]
not sure if this is a bug or something i'm doing wrong, but:

the osu app takes control of my tablet when it starts up (that's fine) but doesn't give it back to the OS when i close the app, and i then need to unplug it and plug it back in to be able to use it

not sure if i'm doing something wrong or if there's a config setting I missed?

**Comment by peppy** [comment]
We can't work with the details you've provided. It does not sound correct, but it's hard to know what you even mean (since we've never heard of this before) without video or otherwise.

**Reply by ghost** [reply]
Sorry about that. I honestly wasn't sure what information would be needed, since this isn't something that can be found in error logs or whatever.

Here's a video: https://www.youtube.com/watch?v=8Hab1Zv6ZjU

In short:
- Tablet works in OS.
- Open osu! lazer app
- Tablet works in osu! app
- Close osu! app
- Tablet is not usable

If I re-open the osu! app after that, I am still not able to use the tablet.

I've been able to reproduce this on a separate computer with a brand-new installation of lazer.

Additionally, if I alt+tab out of osu! without closing it, I can't use the tablet then either, but can use it in osu! when I alt+tab back.

**Reply by peppy** [reply]
Thanks, that helps a lot. Can you also include your tablet model and which driver version you are using?

**Reply by ghost** [reply]
Gaomon S620. How do I find my driver version? The tablet doesn't (seem to) show up in Device Manager, and I haven't installed a driver myself - is there a driver I should be downloading?

**Reply by peppy** [reply]
That's enough information (assuming you have always used the tablet plug and play - never installed any software).

Could you try turning off tablet support in the game and:

- See if the tablet works outside of osu! after doing so

if not, close osu! (with tablet support still disabled) and then replug your tablet and

- See if the tablet works outside of osu! after starting osu!

**Reply by ghost** [reply]
Just tried it. When I disable tablet support, the tablet stops working everywhere, even after closing osu!, but after replugging it, it starts working both inside and outside of osu! simultaneously, and keeps working when I close osu!.

**Reply by peppy** [reply]
@infinityghost can you give any advice here? probably best to open an issue over your side?

**Reply by InfinityGhost** [reply]
This is really weird behavior, on most setups I've seen the manufacturer driver always have priority over OpenTabletDriver.

Disabling tablet support in o!f only disables the tablet data being passed into any of the input handlers, this could be fixed to actually remove the driver instance now or whenever [v0.6](https://github.com/OpenTabletDriver/OpenTabletDriver/milestone/13) gets finished up.

**Reply by peppy** [reply]
@spiralw also, just to confirm, if you have tablet support turned off in osu! from before you start the game, your tablet still works after you start the game correct?

**Reply by ghost** [reply]
it does, yes
